### DISCUSSION TITLE
IE11 invalid argument

### DISCUSSION THREAD

**Discussion by JochemAlb** [boris-jenicek/ngx-awesome-popup#7] [Bug report]
Hi, we'd like to use your asesome popup. However one of the requirements for our app is to also be available in IE11 (unfortunately).
When we import NgxAwesomePopupModule in the app.module.ts file, our app won't work in IE11, we get a blank page. This error is thrown on the console ("Ongeldig argument" means "Invalid argument"):
![image](https://user-images.githubusercontent.com/90708843/133284635-ad19f7cd-dd61-4c15-8ba6-17d5e7ffc534.png)

Other browsers such as chrome and firefox are just fine.
Is IE11 meant to be supported? Is this an issue that can be looked at in a short period of time?

Thank you,
Jochem.

**Comment by boris-jenicek** [comment]
Hi, it's great that you found it useful. I am not sure what to say to IE 11 support. 
I will take a look at it soon.

**Reply by JochemAlb** [reply]
Thank you. 
It would be nice if the toaster could work in IE11, but I think we'd be happy already if this error doesn't give us a blank page anymore. As long as the rest op de app keeps working in IE11.

**Comment by boris-jenicek** [comment]
Thank you @JochemAlb for your input, It took a while to set up an environment on windows but I found the issue and it was animation keyframes that are not supported by IE, I made an exception in the code, and now should work. Please check the new version https://github.com/costlydeveloper/ngx-awesome-popup/releases/tag/1.0.5

by the way, I noticed that IE does not support many other things, for example (append, prepend, remove.. etc) so I installed an additional library in my IE test environment https://www.npmjs.com/package/dom4 and included script in angular.json 
![Screenshot_7](https://user-images.githubusercontent.com/55545296/133861397-dd7575a0-2406-45c5-a0b6-1073d4a21e05.jpg)

**Reply by JochemAlb** [reply]
Thank you @costlydeveloper for looking into this and fixing this that quickly, much appreciated :). It works like a charm.

**Reply by boris-jenicek** [reply]
I am glad it works :) @JochemAlb, and you are welcome, support the ngx-awesome-popup by adding a star to a GitHub repo. Thanks!
